VTEX - Promotions & Taxes API.json

```diff
@@ -5540,7 +5540,75 @@
+                                    {
+                                        "id": 100006808,
+                                        "itemId": 100006808,
+                                        "salesChannel": 0,
+                                        "sellerId": "1",
+                                        "price": 10.0,
+                                        "listPrice": 10.0,
+                                        "validFrom": "2023-01-01T03:00:00",
+                                        "validTo": "2025-01-01T03:00:00",
+                                        "lastUpdate": "2023-12-20T10:58:55.2417",
+                                        "hash": "1000068081000068081101001/01/1900 03:00:0001/01/4000 03:00:0012/20/2023 10:58:55"
+                                    }
+                                ],
+                                "schema": {
+                                    "type" : "array",
+                                    "description": "List of prices.",
+                                    "items":{
+                                        "description": "Object containing information about prices.",
+                                        "type": "object",
+                                        "properties": {
+                                            "id": {
+                                                "type": "integer",
+                                                "description": "Product ID."
+                                            },
+                                            "itemId": {
+                                                "type": "integer",
+                                                "description": "Item ID."
+                                            },
+                                            "salesChannel": {
+                                                "type": "integer",
+                                                "description": "Sales channel."
+                                            },
+                                            "sellerId": {
+                                                "type": "string",
+                                                "description": "Seller ID."
+                                            },
+                                            "price": {
+                                                "type": "number",
+                                                "description": "Price."
+                                            },
+                                            "listPrice": {
+                                                "type": "number",
+                                                "description": "List Price."
+                                            },
+                                            "ValidFrom": {
+                                                "type": "string",
+                                                "description": "Valid from."
```

Comment:
Sugestão: Indicates the date and time when the fixed price will start to be valid.